dc/core/resources/BorderRectangle.qml

```diff
@@ -39,7 +39,7 @@ Item {
         width: Style.borderWidth
         height: Style.borderWidth
         radius: Style.borderWidth
-        color: Style.inactiveBorderColor
+        color: "#00000000"
```

Comment:
It is used to make the circles invisible. I don't know the mechanism... Before it was "color: parent.color" but since borderRect is not a Rectangle anymore but an Item it has no color anymore.
I don't really know how to name that... invisibleBordersColor (LoL) ?? 
inactiveBorderColor is still used at line 61 and it is white, activeBorderColor is yellow.